src/components/Header/Header.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import Navigation from "./Navigation/Navigation";
+import Navigation from "./Navigation";
```

Comment:
1 i 3  linkijka kodu pojedynczy cudzysłów   2 i 4 pojedynczy

Reply:
tu akurat jak webstorm za mnie dopisuje reszte adresu to wrzuca mi w podwojne cudzysłowie zazwyczaj w plikach js uzywam pojedynczych (chyba ze przy wyjatkach) a podwojne w htmlu. Natomiast przy importach nie zwracalbym na to szczegolnie uwagi jezeli piszesz recznie uzywaj pojedynczych.